introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5.ipynb

```diff
@@ -934,6 +934,115 @@
     "    print(f\"{bold} The reasoning result is{unbold}: '{generated_texts}'{newline}\")"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        "max_length": 5000,
</pre><p>Isn't this too large? Does the model even support 5000 tokens?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>It does, i noticed BloomZ can handle such thing. I reduced to 1000.</p>